File: problems/0004_median-of-two-sorted-arrays/concat-sort/Solution.py
#!/usr/bin/python3

class Solution:
    def findMedianSortedArrays(self, nums1: List[int], nums2: List[int]) -> float:
        nums = nums1 + nums2
        nums.sort()
        size = len(nums)
        i = size // 2
        return (nums[i - 1] + nums[i]) / 2 if size % 2 == 0 else nums[i]

File: problems/0004_median-of-two-sorted-arrays/concat-sort/solution.rb
#!/usr/bin/ruby

# @param {Integer[]} nums1
# @param {Integer[]} nums2
# @return {Float}
def find_median_sorted_arrays(nums1, nums2)
    nums = nums1.concat(nums2)
    nums.sort!
    i = nums.length / 2
    return (nums.length % 2 == 0 ? (nums[i - 1] + nums[i]) / 2.0 : nums[i])
end
